packages/elements/src/icon/index.ts

```diff
@@ -112,13 +124,22 @@ export class Icon extends BasicElement {
     }
   }
 
+  /**
+   * Check if the autosuggest has content
+   * @returns content exists
+   */
+  private get iconMap (): string | null {
+    return this.icon && this.config?.icon?.map[this.icon] || null;
```

Comment:
This looks too interesting to be good.
Is there a way we can avoid making use of all those optional chaining?